.github/workflows/bump_dependencies.yml

```diff
@@ -43,6 +44,7 @@ jobs:
           title: Automated Go Dependencies Update
           body: This is an auto-generated PR to update go dependencies. Please confirm tests are passing before merging.
           reviewers: JohnN193
+          draft: always-true
```

Comment:
interesting, what does this do?

Reply:
whenever this workflow makes a pr or updates a pr, it should set that pr to be in draft mode. hoping to use this to be able to trigger tests to run in the pr without having to create a different pr, though i might need to make another change to other workflows first before it actually works